### ISSUE TITLE
Need some accessibility labels to improve VoiceOver reading

### ISSUE THREAD

**Issue by aquilent-dgemoets** [USDepartmentofLabor/Child-Labor#57]
An accessibility labels tells VoiceOver to read a given section of text in a certain way.  Two specific examples come to mind where we should use this:
- Everywhere that the text includes "No." (e.g., "No. of Investigators), the accessibility label should be set (e.g., "Number of Investigators").
- Referral Mechanism Exists btwn Authorities and Social Services (Enforcement screen, both tabs):  accessibility label should be, "Referral Mechanism Exists between Authorities and Social Services".

Any additional examples where we need accessibility labels can be added to this Issue.

**Comment by aquilent-dgemoets** [comment]
These are fixed in the latest beta (9/9/16).
However, reading of territory names with abbreviations is not optimal (e.g., FBiH, RS).  We will try to address this in the final version.

**Comment by aquilent-dgemoets** [comment]
To elaborate:
•	It seems to reads all territories, then reads all values; it should read name/value, name/value, etc., (e.g., "FBiH, yes, RS, yes, BD, yes")
•	Sometimes its associating the wrong value with a given territory
•	Examples
	•	Pakistan, Legal Standards:  "Work" sections reading in wrong order:  Federal, KP, KP's value, Federal's value.
	•	Iraq, Enforcement (both tabs).  These all need to be audited for correct tabbing/reading order.
•	This might only happen in iOS 9.x?

**Comment by aquilent-dgemoets** [comment]
This all appears to be fixed in the latest beta.  Tested on an iPhone running iOS 10.0.2.

**Comment by jfc3-dol** [comment]
While reviewing the changes over here at DOL before sending them to Apple I noticed the footnotes over lap content and themselves on the Iraq "Legal Standards" screen. There might be other screens this happens on. We are using an iPod (5th Generation) Model ME643LL/A which is running iOS 9.3.4.

Please see screenshot.

![ilab -iraq - legal standards footnotes over lap content and themselves](https://cloud.githubusercontent.com/assets/10423487/20183767/6cf8856a-a734-11e6-87e4-0a57b168d4db.PNG)

You might want to open another issue for this or add it to an already open one.

**Comment by aquilent-dgemoets** [comment]
I've run it with an iPhone 5 simulator running iOS 9.3 with the exact same 1136-by-640-pixel resolution as an iPod 5th generation, but can't duplicate the problem.  I guess it's an iPod Touch-specific problem.  In other words, this isn't likely to be able to be fixed.  :-(

**Comment by jfc3-dol** [comment]
Not a problem, I thought I would mention it in case it was an issue for all devices. I looked at the screen on our even older iPad in bot 1X and 2X mode and the screen looks as you would expect it. So it must be a device thing then.

**Comment by carolynhuang** [comment]
Please clarify whether this is still an outstanding issue.

**Comment by jfc3-dol** [comment]
Seems to be fixed so closing the issue.
